src/ci_providers/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as providerAppveyorci from './provider_appveyorci'
```

Comment:
```suggestion
export * from './provider_appveyorci'
```

Reply:
Am I correctly understanding this suggestion is not valid?

Reply:
You are correct. Sorry for not making that clear.